src/index.css

```diff
@@ -0,0 +1 @@
+body { margin: 0; padding: 0; }
```

Comment:
You shouldn't have css files on a grommet application.

Reply:
In the index.js file an 8px margin is added around the application. I wasn't sure how else to get rid of this margin since it is happening outside of `<Grommet> ... </Grommet>`

Reply:
It will be better to inline the css on the HTML body and remove the CSS file.